### PR TITLE
Move from a dict to properties for static values in virt.

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

File: shakenfist/virt.py [hunk 1]
```diff
@@ -170,47 +178,203 @@ def from_db(uuid):
...
+
+    @uuid.setter
+    def uuid(self, _):
+        raise exceptions.NotImplementedException(
+            'You cannot set a static variable')
+
+    @uuid.deleter
+    def uuid(self):
+        raise exceptions.NotImplementedException(
+            'You cannot delete a static variable')
```
Comment (mandoonandy) [inline shakenfist/virt.py:195]: I think you could reduce code by not defining the `setter` and `deleter`. If they are called I think you get an `AttributeError` which is probably more pythonic.

**Review by mandoonandy — COMMENTED** [review]
Looks very pretty to me! (Just one question)

File: shakenfist/virt.py (hunk 1 above)
Comment (mikalstill) [reply shakenfist/virt.py:195]: Oh interesting. I will test this when I am not in a meeting.

Comment (mikalstill) [reply shakenfist/virt.py]: And you're right. Fixed.

**Review by mandoonandy — APPROVED** [review]
LGTM

**Pull request merged**